sql-database/transparent-data-encryption/setup-tde-byok-sqlmi.ps1

```diff
@@ -31,19 +29,22 @@ Update-AzKeyVaultNetworkRuleSet -VaultName "MyKeyVault" -DefaultAction Deny
 
 # 2. Provide TDE Protector key (skip if already done)
 
-# Generate new key directly in Azure Key Vault (recommended for test purposes only - uncomment below):
-# $key = Add-AzureKeyVaultKey -VaultName MyKeyVault -Name MyTDEKey -Destination Software -Size 2048
+# The recommended way is to import an existing key from a .pfx file:
+$keypath = "c:\some_path\mytdekey.pfx" # Supply your desired path and name
+$securepfxpwd = ConvertTo-SecureString -String (New-Guid).Guid -AsPlainText -Force 
```

Comment:
This is my mistake, because this is loaded from an existing PFX rather than creating a new one. Rather than generating or providing a password here, there should be some kind of requirement for a user-supplied value for the PFX. This can be in the form of user input or a string value which indicates the value is user-supplied such as a string with the contents `"<PFX private key password>"`

Reply:
Done.

![image](https://user-images.githubusercontent.com/40007119/55821187-bf7f1a00-5ab1-11e9-935a-d589ae32184c.png)